tests/fixtures/entity-viewer/transcript.ts

```diff
@@ -35,7 +35,7 @@ import { TranscriptMetadata } from 'src/shared/types/thoas/metadata';
 type ProteinCodingProductGeneratingContext = Omit<
   FullProductGeneratingContext,
   'cds'
-> & { cds: FullCDS };
+> & { cds: FullCDS | null };
```

Comment:
If a product-generating context produces a protein product, as the name of the type suggests it does, its cds will not be null. CDS is null for a non-coding product.

Reply:
So does that mean i have to create a different type for noncoding transcript then?